examples/orders/create_payment.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @docs https://docs.mollie.com/reference/v2/orders-api/create-order-payment
+ */
+const { createMollieClient } = require('@mollie/api-client');
+
+const mollieClient = createMollieClient({ apiKey: 'test_dHar4XY7LxsDOtmnkVtjNVWXLSlXsM' });
+
+(async () => {
+  try {
+    const payment = await mollieClient.orders_payments.update({
```

Comment:
This should be `mollieClient.orders_payments.create`

Reply:
ow oops... I'll change it in a few minutes